src/main/java/sirius/tagliatelle/Tagliatelle.java

```diff
@@ -418,8 +418,8 @@ private void writeWarningsToLog(CompilationContext compilationContext) {
         message.append("Warnings when compiling ")
                .append(compilationContext.getTemplate().getShortName())
                .append(":\n");
-        compilationContext.getErrors().forEach(message::append);
-        message.append(" - Template: ");
+        compilationContext.getErrors().forEach(error -> message.append(error).append("\n\n"));
```

Comment:
Why 2 line breaks?

Reply:
readability - could also be done with a single break - suggestions?

Reply:
Readability is better with two line breaks 👍